src/libs/ReportUtils.js

```diff
@@ -460,6 +449,23 @@ function isConciergeChatReport(report) {
     return lodashGet(report, 'participantAccountIDs', []).length === 1 && Number(report.participantAccountIDs[0]) === CONST.ACCOUNT_ID.CONCIERGE && !isChatThread(report);
 }
 
+/**
+ * Check if report is a DM and personal detail of participant is optimistic data
+ * @param {String} report
```

Comment:
Also this is wrong, should be this, right?

```suggestion
 * @param {Object} report
 * @param {Array<Number>} report.participantAccountIDs
```